charts/mlrun-ce/templates/jupyter-notebook/ingress.yaml

```diff
@@ -17,7 +17,6 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  ingressClassName: nginx
```

Comment:
Why is this removed?
What is the default `ingressClassName` that will be used?

Reply:
Fixed